### PR TITLE
cherry-pick dockerfile, docker-compose, actions, and Makefile adjustments onto v5

### PR DESCRIPTION
**Pull request by i-norden**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,11 +5,15 @@ services:
     restart: on-failure
     depends_on:
       - ipld-eth-db
+    # Use local build
+    #build:
+      #context: .
+      #dockerfile: Dockerfile
```
Comment (roysc) [inline docker-compose.yml:11]: leftovers?

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -10,32 +10,38 @@ jobs:
...
+  #    - name: Docker Login to Github Registry
+  #      run: echo ${{ secrets.GITHUB_TOKEN }} | docker login https://docker.pkg.github.com -u vulcanize --password-stdin
+  #    - name: Docker Pull
+  #      run: docker pull docker.pkg.github.com/vulcanize/ipld-eth-db/ipld-eth-db:${{steps.vars.outputs.sha}}
+  #    - name: Docker Login to Docker Registry
+  #      run: echo ${{ secrets.VULCANIZEJENKINS_PAT }} | docker login -u vulcanizejenkins --password-stdin
+  #    - name: Tag docker image
+  #      run: docker tag docker.pkg.github.com/vulcanize/ipld-eth-db/ipld-eth-db:${{steps.vars.outputs.sha}} vulcanize/ipld-eth-db:${{steps.vars.outputs.tag}}
+  #    - name: Docker Push to Docker Hub
+  #      run: docker push vulcanize/ipld-eth-db:${{steps.vars.outputs.tag}}
```
Comment (roysc) [inline .github/workflows/publish.yaml]: can we just drop this stuff? seems that it was just left for reference

**Review by roysc — APPROVED** [review]
LGTM

File: docker-compose.yml (hunk 1 above)
Comment (roysc) [reply docker-compose.yml:11]: Oh I see, this was also left for reference :)

File: .github/workflows/publish.yaml (hunk 2 above)
Comment (i-norden) [reply .github/workflows/publish.yaml]: Yeah good idea, dropping this then merging. Thanks!

**Pull request merged**